### ISSUE TITLE
Cannot tab to previous step button in MultiStepDialog

### ISSUE THREAD

**Issue by bvandrc** [palantir/blueprint#6057]
#### Environment

- __Package version(s)__: Latest
- __Operating System__: Windows
- __Browser name and version__: Chrome

#### Steps to reproduce

1. Go to a multistep dialog step greater than 1 (ie at https://blueprintjs.com/docs/#core/components/dialog.multistep-dialog, then go to step 2)
2. Cannot tab to a previous step, but can click it to take effect

#### Actual behavior

Cannot tab to the step to go back.

#### Expected behavior

Can tab to step to go back, and hit enter to take effect

#### Possible solution

Set `tabIndex={0}` to the `div` that has the `onClick` for changing the dialog step so it can be tabbed to. May also need to add `onKeyDown` for `enter` keypress to take effect.
